src/index.js

```diff
@@ -0,0 +1,229 @@
+const inquirer = require("inquirer");
+
+const Db = require("./connection/connection");
+
+const {
+  generateDepartmentChoices,
+  generateRoleChoices,
+  generateEmployeeChoices,
+} = require("./helper/util");
+
+const {
+  menu,
+  addDepartment,
+  sectionToUpdate,
+  updateEmployeeName,
+  constructRoleQuestions,
+  constructDepartmentQuestions,
+  constructEmployeeQuestions,
+} = require("./questions/questions");
+
+//create a new instance of the Db class to use the start , stop and query methods
+const db = new Db({
+  host: process.envDB_HOST || "localhost",
```

Comment:
Consider moving the `dbOptions` into the Db class and only pass in the database. This will make it easier when reusing the class.